packages/odyssey-react-mui/src/labs/DataView/DataCard.tsx

```diff
@@ -110,7 +110,7 @@ const CardContentContainer = styled("div", {
 const CardChildrenContainer = styled("div", {
   shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
 })<{ odysseyDesignTokens: DesignTokens }>(({ odysseyDesignTokens }) => ({
-  ["div + &"]: {
+  ["&:not(:first-child)"]: {
```

Comment:
What is this meant to do? If the consumer is passing in children, I think we should let them control the margin/padding of those children right?

Reply:
The `div` here would get the margin, then each `p` would also have the same margin right?
```html
<div>
<p>this will have margin</p>
<p>this will have the same margin</p>
</div>
```

Reply:
No, only the div — it's `&:not(:first-child)`, not `& *:not(:first-child)`